__tests__/universal/routes.spec.jsx

```diff
@@ -44,17 +47,37 @@ describe('routes', () => {
 
   beforeEach(() => jest.clearAllMocks());
 
-  it('should set up the root module route first', () => {
-    const RootRoute = createRoutes(store)[0];
-    expect(ReactTestUtils.isElement(RootRoute)).toBe(true);
-    expect(RootRoute.props).toEqual({ moduleName: 'fakeRootModule', store });
+  describe('RootModule without childroutes', () => {
+    it('should add a path on the root route', () => {
+      mockHasChildRoutesValue = false;
+      const RootRoute = createRoutes(store)[0];
```

Comment:
consider asserting the lenght of the array returned by createRoutes.
As it stands, if we modify the code to include a second route, none of these tests will fail